Integrations/CounterCraft/CounterCraft.yml

```diff
@@ -0,0 +1,420 @@
+      description: Event date
+      type: date
+    - contextPath: CounterCraft.Event.HostName
+      description: Host name
+      type: string
+    - contextPath: CounterCraft.Event.ServiceName
+      description: Service name
+      type: string
+    - contextPath: CounterCraft.Event.TypeCode
+      description: Type
+      type: string
+    - contextPath: CounterCraft.Event.Score
+      description: Score
+      type: number
+    - contextPath: CounterCraft.Event.Tags
+      description: Tags
+      type: string
+    - contextPath: CounterCraft.Events.Data
+      description: Data
+      type: unknown
+    description: Get full list of Events
+  - name: countercraft-create-campaign
+    arguments:
+    - name: name
+      required: true
+      description: Campaign name
+    - name: description
+      required: true
+      description: Campaign description
+    outputs:
+    - contextPath: CounterCraft.Campaign.ID
+      description: Campaign ID
+      type: number
+    - contextPath: CounterCraft.Campaign.Name
+      description: Name
+      type: string
+    - contextPath: CounterCraft.Campaign.Description
+      description: Description
+      type: string
+    - contextPath: CounterCraft.Campaign.StatusCode
+      description: Status Code
+      type: string
+    description: Create a new deception campaign
+  - name: countercraft-list-dsns
+    arguments: []
+    outputs:
+    - contextPath: CounterCraft.DSN.ID
+      description: ID
+      type: number
+    - contextPath: CounterCraft.DSN.Name
+      description: Name
+      type: string
+    - contextPath: CounterCraft.DSN.Description
+      description: Description
+      type: string
+    - contextPath: CounterCraft.DSN.Hostname
+      description: Hostname
+      type: string
+    - contextPath: CounterCraft.DSN.Port
+      description: Port
+      type: number
+    description: List Deception Support Nodes (DSNs)
+  - name: countercraft-list-providers
+    arguments: []
+    outputs:
+    - contextPath: ID
+      description: ID
+      type: number
+    - contextPath: CounterCraft.Provider.Name
+      description: Name
+      type: string
+    - contextPath: CounterCraft.Provider.Description
+      description: Description
+      type: string
+    - contextPath: CounterCraft.Provider.TypeCode
+      description: Type
+      type: string
+    - contextPath: CounterCraft.Provider.StatusCode
+      description: Status
+      type: string
+    description: 'List providers (providers for hosts or services i.e. AWS or Office365) '
+  - name: countercraft-create-host-machine
+    arguments:
+    - name: name
+      required: true
+      description: Name
+    - name: description
+      description: Description
+    - name: provider_id
+      required: true
+      description: Provider
+    - name: deception_support_node_id
+      required: true
+      description: Deception Support Node ID
+    - name: campaign_id
+      required: true
+      description: Campaign
```

Comment:
please elaborate how can this be retrieved

Reply:
Same than above. If I have added an example in the documentation, do I need to add it to the description field?

Reply:
Done